concepts/arrays/about.md

```diff
@@ -126,33 +135,23 @@ let mut a = ArrayTrait::new();
 }
 ```
 
-# Removing Array Elements
+When using the subscripting operator arr[index] instead of arr.at(index), it's essential to explicitly specify the element type of the array. Failure to do so will result in compilation errors.
```

Comment:
This might have been true before, but is no longer the case. The following will now compile:
```rust
    let mut a = ArrayTrait::new();
    a.append(0);
    a.append(1);

    let first = a[0];
    let second = a[1];
```
Update the sentence to only introduce the example that follows it.